packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -80,7 +85,14 @@ const Row = ({
             className={
                 classname_for_demo_and_eu ??
                 classNames(`cfd-accounts-compare-modal__table-row${pre_appstore_class}`, {
+                    'cfd-accounts-compare-modal__table-row--other-country': is_af || is_mg || is_pk,
                     [`cfd-accounts-compare-modal__table-row--leverage${pre_appstore_class}`]: is_leverage_row,
+                    'cfd-accounts-compare-modal__table-row--leverage__af': is_leverage_row && is_af,
+                    'cfd-accounts-compare-modal__table-row--instruments__af': is_instruments_row && is_af,
+                    'cfd-accounts-compare-modal__table-row--leverage__pk': is_leverage_row && is_pk,
+                    'cfd-accounts-compare-modal__table-row--instruments__pk': is_instruments_row && is_pk,
+                    'cfd-accounts-compare-modal__table-row--leverage__mg': is_leverage_row && is_mg,
+                    'cfd-accounts-compare-modal__table-row--instruments__mg': is_instruments_row && is_mg,
```

Comment:
can we simplify this? 


`const is_other_countries =   ['af','pk','mg'].includes(client_country);`

and in styles here , 
```
 'cfd-accounts-compare-modal__table-row--other-country': is_other_countries,
 [`cfd-accounts-compare-modal__table-row--leverage__${client_country}`] : is_leverage_row && is_other_countries,
 [`cfd-accounts-compare-modal__table-row--instruments__${client_country}`] : is_instruments_row && is_other_countries,
```

will this work?

Reply:
Yes, thanks.
Refactored the code